### ISSUE TITLE
Broken Cryptpad after 1.15.0 upgrade

### ISSUE THREAD

**Issue by croulibri** [YunoHost-Apps/cryptpad_ynh#17]
After updating the app to 1.15.0, I can open cryptdrive
But when I  try to open a pad (text) I get 

> Cannot GET /pad/undefined/pad/inner.html?ver=1.12.0

The same error appears with the editing or the read only link.

I just install cryptpad with yunohost package 5 days ago and this is my first upgrade.
I did upgrade with `yunohost app upgrade --verbose -u https://github.com/YunoHost-Apps/cryptpad_ynh cryptpad`

During upgrade, I received few warnings :

>Attention : 2017-09-14 18:57:35 URL:https://codeload.github.com/xwiki-labs/cryptpad/tar.gz/v1.15.0 [7890094] -> "app.tar.gz" [1]
>Attention : npm WARN cryptpad@1.15.0 No repository field.
>Attention : npm WARN cryptpad@1.15.0 No license field.
>Attention : 
>up to date in 3.203s
>Attention : bower bootstrap#~3.1.1              invalid-meta The "main" field cannot contain font, image, audio, or video files
>Attention : bower bootstrap#~3.1.1              invalid-meta The "main" field cannot contain font, image, audio, or video files
>Attention : bower bootstrap#~3.1.1              invalid-meta The "main" field cannot contain font, image, audio, or video files
>Attention : bower bootstrap#~3.1.1              invalid-meta The "main" field cannot contain font, image, audio, or video files

How can I repair my CryptPad install ?

**Comment by croulibri** [comment]
I tried to upgrade again ... no change, still broken text, spreadsheet and slide pads.
I uninstalled the app then restore a backup.
Miracle... everything works again :-)
No idea what happened, but I realized that the situation before the upgrade was not clean either : I had all the time a pop-up about the "1.12 update" when the Cryptpad version was the 1.14.

Anyway, I can enjoy again my yunocryptpad !
